src/routes/Project/containers/ProjectParamsContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+
+import ProjectParams from '../components/ProjectParams';
+import { updateTempo, updateTimeSig, updateTimeDiv } from '../actions/TableActions';
+
+const mapDispatchToProps = {
+  updateTempo,
+  updateTimeSig,
+  updateTimeDiv
```

Comment:
I'm ok with it just this once if you agree not to ever do it again. If we have agreed that the state will be synced we should stick to not mixing local state and synced state. later on this will become cumbersome.